File: C_Sharp_Course/C_Sharp_Course/Files/static_class_attributes.cs
using C_Sharp_Course.Classes;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.CompilerServices;
using System.Text;
using System.Threading.Tasks;

namespace C_Sharp_Course
{
    class UsingStaticClassAttributes
    {
        static void StaticClassAttributes(string[] args)
        {
            Song holiday = new Song("Holiday", "Green Day", 200);
            Console.WriteLine(holiday.title);
            Console.WriteLine(holiday.artist);
            Console.WriteLine(Song.songCount);
            Console.WriteLine(holiday.GetSongCount());

            Console.WriteLine();

            Song kashmir = new Song("Kashmir", "Led Zeppelin", 150);
            Console.WriteLine(kashmir.title);
            Console.WriteLine(kashmir.artist);
            Console.WriteLine(Song.songCount);
            Console.WriteLine(kashmir.GetSongCount());

            Console.WriteLine();

            Console.ReadLine();
        }
    }
}

File: Java_Course/src/App/StaticClassAttributes.java
package App;

public class StaticClassAttributes {
	public static void staticClassAttributes(String [] args) {
		
		Song holiday = new Song("Holiday", "Green Day", 200);
		Song americanIdiot =  new Song("American Idiot", "Green Day", 168);
		
		System.out.println(holiday.getSongsCount());
		System.out.println(americanIdiot.getSongsCount());
		
		Song africa = new Song("Africa", "Toto", 300);
		
		System.out.println(holiday.getSongsCount());
		System.out.println(americanIdiot.getSongsCount());
		System.out.println(africa.getSongsCount());
	}
}
